From 6c51c3e81be968129d798eff02c26fd5a7df2388 Mon Sep 17 00:00:00 2001
From: Ryan May <rmay@ucar.edu>
Date: Wed, 5 May 2021 12:42:10 -0600
Subject: [PATCH 1/4] Build without USE_WIN32_FILEIO

This keeps things consistent with current conda-forge builds of tiff
(and poppler).
---
 setup.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/setup.py b/setup.py
index 5ecd6b816..3fef7ae15 100644
--- a/setup.py
+++ b/setup.py
@@ -871,12 +871,6 @@ class pil_build_ext(build_ext):
         if feature.get("tiff"):
             libs.append(feature.get("tiff"))
             defs.append(("HAVE_LIBTIFF", None))
-            if sys.platform == "win32":
-                # This define needs to be defined if-and-only-if it was defined
-                # when compiling LibTIFF. LibTIFF doesn't expose it in `tiffconf.h`,
-                # so we have to guess; by default it is defined in all Windows builds.
-                # See #4237, #5243, #5359 for more information.
-                defs.append(("USE_WIN32_FILEIO", None))
         if feature.get("jpeg"):
             libs.append(feature.get("jpeg"))
             defs.append(("HAVE_LIBJPEG", None))
